includes/admin/load.php

```diff
@@ -31,11 +29,8 @@ function perflab_add_features_page() {
 		add_action( "load-{$hook_suffix}", 'perflab_load_features_page', 10, 0 );
 		add_filter( 'plugin_action_links_' . plugin_basename( PERFLAB_MAIN_FILE ), 'perflab_plugin_action_links_add_settings' );
 	}
-
-	return $hook_suffix;
 }
 
-// @phpstan-ignore-next-line
```

Comment:
As noted in https://github.com/WordPress/performance/pull/1188#discussion_r1589529447 this returning of the `$hook_suffix` was only being used in tests. This PR eliminates the need for that return value by instead computing the `$hook_suffix` in the test, and then seeing if the expected action is added.